src/Arcus.EventGrid.Tests.Integration/Arcus.EventGrid.Tests.Integration.csproj

```diff
@@ -1,10 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2;netcoreapp3.0</TargetFrameworks>
```

Comment:
Are our libs, other than testing, using .NET Standard? If so, fine

Reply:
The Arcus.EventGridSecurity is `<TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>`

Reply:
👌